pkg/gds/emails/client.go

```diff
@@ -485,3 +486,40 @@ func (m *EmailManager) SendReissuanceStarted(vasp *pb.VASP, whisperLink string)
 
 	return sent, errs.ErrorOrNil()
 }
+
+// SendReissuanceAdminNotification sends the admins a notice that an identity certificate
+// has been reissued. This allows the admins that the reissuance has been done automatically
+func (m *EmailManager) SendReissuanceAdminNotification(vasp *pb.VASP, reissueDate time.Time) (sent int, err error) {
+	// Create the template context
+	ctx := ReissuanceAdminNotificationData{
```

Comment:
This is a stickler thing, but can we rename this variable? `ctx` usually refers to the go `context.Context` throughout the rest of the codebase so it could be minorly confusing.

Reply:
It looks like `ctx` variable is being used in the rest of client.go for email data. should we open a new story for this change?